core/src/it/scala/org/scalawag/jibe/backend/ubuntu/IsPackageInstalled.scala

```diff
@@ -0,0 +1,27 @@
+package org.scalawag.jibe.backend.ubuntu
+
+import org.scalatest.{FunSpec, Matchers}
+import org.scalawag.jibe.mandate.command.IsPackageInstalled
+
+class IsPackageInstalled extends FunSpec with Matchers with VagrantTest {
+
+  it("non-existant packages do not need to be installed") {
+    commander.execute(IsPackageInstalled("packagedoesntexist")) shouldBe true // true = 0
+  }
+
+  it("empty package name does not need to be installed") {
+    commander.execute(IsPackageInstalled("")) shouldBe true // true = 0
+  }
+
+  it("vim does not need to be installed when it is already there") {
+    rootSsh.exec(log, s"apt-get install --assume-yes vim")
+    commander.execute(IsPackageInstalled("vim")) shouldBe true // true = 0
+  }
+
+  it("vim does need to be installed when it is not already there") {
+    rootSsh.exec(log, s"apt-get remove --assume-yes vim")
+    commander.execute(IsPackageInstalled("vim")) shouldBe false // false = 1
+  }
+
```

Comment:
You need more test cases here:
 - the upgrade-needed cases
   - install old one, ask about no version Package
   - install old one, ask about newer version Package
 - the downgrade-needed case
   - install newish one, ask about older version Package
 - more true cases
   - install old version, ask for that version